db/seeds.d/035-admin.rb

```diff
@@ -42,11 +42,13 @@
 end
 
 # First real admin user account
-unless User.unscoped.only_admin.except_hidden.present?
```

Comment:
`only_admin` doesn't find the exising admin user when upgrading due to the changed format. I've made this seed more sensitive so it first checks if a user with the admin login exists, which will also be good so it doesn't fail when trying to seed an existing user name.